### PR TITLE
Add OVR Toolkit wrist macros as an option for triggering Reset Bindings

### PR DESCRIPTION
**Pull request by nullstalgia**
Hello again :)

I wasn't the biggest fan of using up precious controller buttons for the bindings (especially if it conflicts with other games), and OVR Toolkit has had Macro support for a good few months now. So I wanted to share my current setup with others!

Demo Gif:

ezgif com-gif-maker(14)

I also yoinked the new SVG icons from the new GUI and made the arrows thicker for readability, these custom icons are provided in the instructions.
quickresetBold

resetBold

Unfortunately, until [SlimeVR-Server Issue #254](https://github.com/SlimeVR/SlimeVR-Server/issues/254) is resolved, an [AutoHotKey script](https://gist.github.com/nullstalgia/18595feddac83ad8d6d8e0e8117e64d1) is required to add a minor delay to the binding, as otherwise your head and arms will not be in a favorable position for a reset.

### CODE REVIEW

File: server-setup/setting-reset-bindings.md [hunk 1]
```diff
@@ -113,6 +113,33 @@ Make sure OVR Advanced Settings is closed before following these steps or you wi
...
+8. Repeat steps 5-7 for the other binding.
+9. Click "Save Changes", and then check the Macros tab on your OVR Toolkit wristwatch for your new bindings! (You may need to restart OVR Toolkit!)
+
+Example GIF of adding the Quick Reset binding:
+
+![img](..\assets\img\ovrtMacro.gif)
+
+#### Warning
+
+Currently the SlimeVR server resets the positions the instant the binding is pressed, which is not ideal if you are looking down at your extended arms. However, this can be overcome with a simple AutoHotKey script until a configuration option is added to add a countdown to those bindings. [AutoHotKey Script Here](https://gist.github.com/nullstalgia/18595feddac83ad8d6d8e0e8117e64d1)
```
Comment (TheDevMinerTV) [inline server-setup/setting-reset-bindings.md:141]: It would be cool if you could add the Gist's code here directly in a code block

Comment (Eirenliel) [reply server-setup/setting-reset-bindings.md:141]: Agree with Anna

Comment (nullstalgia) [reply server-setup/setting-reset-bindings.md:141]: Yep, I can absolutely do that! I just was not sure what would look best. If you guys don't edit it first, I'll get to it soon

Comment (nullstalgia) [reply server-setup/setting-reset-bindings.md]: Done! I'll let one of you guys resolve this.

**Review by TheDevMinerTV — APPROVED** [review]

**Review by calliePepper — APPROVED** [review]
This looks great

**Pull request merged**